packages/react-components/react-breadcrumb/docs/Spec.md

```diff
@@ -2,54 +2,479 @@
+| overflowIndex     | number                                  | 1             | Index of item where BreadcrumbMenu should be shown                                       |
+| size              | `small`, `medium`, `large`              | `medium`      | Defines size of the Breadcrumb                                                           |
+
+### BreadcrumbItem
+
+#### Anatomy
+
+![visual anatomy of the BreadcrumbItem component](./assets/breadcrumb-item-anatomy.png)
+
+BreadcrumbItem can be:
+
+- Button
+- Link
+- Non-clickable content
+- Dropdown Menu
+- Icon (optional)
+
+It can contain a tooltip.
+
+#### DOM
+
+Non-clickable element
+
+```HTML
+<li>
+  {children}
+</li>
+```
+
+Link
+
+```HTML
+<li>
+  <a href="#">
+    {children}
+  </a>
+</li>
+```
+
+Button
+
+```HTML
+<li>
+  <button>
+    {children}
+  </button>
+</li>
+```
+
+Usage
+
+```jsx
+<BreadcrumbItem>
+  Item 1
+</BreadcrumbItem>
+<BreadcrumbItem onClick={() => console.log('smth...')}>
+  Item 2
+</BreadcrumbItem>
+<BreadcrumbItem href="#">
+  Item 3
+</BreadcrumbItem>
+```
+
+#### API
+
+| Property     | Values                                  | Default   | Purpose                                     |
+| ------------ | --------------------------------------- | --------- | ------------------------------------------- |
+| active       | boolean                                 | false     | Indicates if the item is active             |
+| disabled     | boolean                                 | false     | Disables Breadcrumb item                    |
+| divider      | _slot_                                  | `chevron` | Sets type of divider                        |
+| focusMode    | `off`, `no-tab`, `tab-exit`, `tab-only` | `off`     | Sets the focus behavior for the Breadcrumb. |
+| icon         | _slot_                                  |           | Sets icon                                   |
+| iconPosition | `before`, `after`                       | `before`  | Sets icon position                          |
+| tooltip      | _slot_                                  | undefined | Sets custom tooltip                         |
+
+#### Breadcrumb icon
+
+An icon can be SVG or image.
+The icon might be added as a prop:
+
+```jsx
+<BreadcrumbItem icon={<IconComponent />}>Item 1</BreadcrumbItem>
+<BreadcrumbItem icon="../path/name.png">Item 1</BreadcrumbItem>
+<BreadcrumbItem>
+  <SomeSVG />
+  Item 1
+</BreadcrumbItem>
+```
+
+### BreadcrumbDivider
+
+#### DOM
+
+```HTML
+<li aria-hidden="true">
+  {children}
+</li>
+```
+
+#### API
+
+| Property | Values                               | Default   | Purpose              |
+| -------- | ------------------------------------ | --------- | -------------------- |
+| divider  | `chevron`, `slash`, custom component | `chevron` | Sets type of divider |
+
+### BreadcrumbMenu
```

Comment:
Why isn't JSX composition good enough for menus in breadcrumbs? I don't see a usage example in the spec so I can only take an assumption right now:

```tsx

<Breadcrumb>
  <BreadcrumbItem>Foo</BreadcrumbItem>
  <Menu>
    <MenuTrigger><BreadcrumbItem>Bar</BreadcrumbItem></MenuTrigger>
   <MenuPopover>{/* rest*/ }</MenuPopover>
  </Menu>
</Breadcrumb>
``

Reply:
That's another question I have, that's why I wanted some early feedback. I see that in a lot of components, partners should implement everything themselves using existing components. Correct me if I'm wrong. I was going to add logic for overflow items to the component because it's a common scenario. But from what I learned from other components, I'll just need an example of how partners should do this themselves

Reply:
Resolving this as we agreed that partners will use JSX composition